xml/x509_subjects.xml

```diff
@@ -475,7 +483,7 @@
   </fingerprint>
 
   <fingerprint pattern="^CN=([a-zA-Z0-9\.\-\_]+),OU=VMware ESX Server Default Certificate,O=VMware\\, Inc,L=Palo Alto,ST=California,C=US$">
-    <description>VMware ESX</description>
+    <description>VMWare ESX</description>
```

Comment:
This PR will likely be closed, but note for self - this change is incorrect. `VMware` is correct.